ssl/ssl_ciph.c

```diff
@@ -1095,6 +1095,10 @@ static int ssl_cipher_process_rulestr(const char *rule_str,
                     && (ca_list[j]->name[buflen] == '\0')) {
                     found = 1;
                     break;
+                } else if (ca_list[j]->stdname && strncmp(buf, ca_list[j]->stdname, buflen) == 0
```

Comment:
Please change `ca_list[j]->stdname` to `ca_list[j]->stdname != NULL`

Reply:
Fixed